### PR TITLE
Added info about docker rm -v

### LINKED ISSUE #1039: Document that docker rm -v is supported
Per https://github.com/vmware/vic/issues/6262, implemented in PR https://github.com/vmware/vic/pull/6680.

### PR DESCRIPTION
**Pull request by alextopuzov**
Fixes #1039

### CODE REVIEW

File: docs/user_doc/vic_app_dev/container_operations.md [hunk 1]
```diff
@@ -54,7 +54,9 @@
...
 |`restart`|[Restart a container](https://docs.docker.com/engine/reference/commandline/restart/)|Yes, since 1.0|
-|`rm`|[Remove a container](https://docs.docker.com/engine/reference/commandline/rm/)|Yes, since 1.0. Removes associated anonymous and regular volumes. Supports the `--force` option and the `name` parameter.  Does not support `docker rm -v`. To view volumes attached to a container that is removed, use `docker volume ls` and `docker volume inspect <id>`. If you continually invoke `docker create` to make more anonymous volumes, those volumes are left behind after each subsequent removal of that container.|
+|`rm`|[Remove a container](https://docs.docker.com/engine/reference/commandline/rm/)|Yes, since 1.0. Removes associated anonymous and regular volumes. Supports the `--force` option and the `name` parameter.  Does not support `docker rm -v`. To view volumes attached to a container that is removed, use `docker volume ls` and `docker volume inspect <id>`. If you continually invoke `docker create` to make more anonymous volumes, those volumes are left behind after each subsequent removal of that container.
```
Comment (stuclem) [inline docs/user_doc/vic_app_dev/container_operations.md]: Remove "Does not support `docker rm -v`."

Comment (alextopuzov) [reply docs/user_doc/vic_app_dev/container_operations.md]: Removed.

File: docs/user_doc/vic_app_dev/container_operations.md [hunk 2]
```diff
@@ -54,7 +54,7 @@
...
+|`rm`|[Remove a container](https://docs.docker.com/engine/reference/commandline/rm/)|Yes, since 1.0. Removes associated anonymous and regular volumes. Supports the `--force` option and the `name` parameter. To view volumes attached to a container that is removed, use `docker volume ls` and `docker volume inspect <id>`. If you continually invoke `docker create` to make more anonymous volumes, those volumes are left behind after each subsequent removal of that container. <br>Since 1.3 `docker rm -v` is also supported. Running the command removes the container and any anonymous volumes joined. If an anonymous volume is in use by another container, it is not removed. Named volumes (specified by name in the create/run command) are not deleted.|
```
Comment (anchal-agrawal) [inline docs/user_doc/vic_app_dev/container_operations.md]: > Removes associated anonymous and regular volumes.

This is inaccurate. Just `docker rm` does not remove any volumes.

Comment (anchal-agrawal) [inline docs/user_doc/vic_app_dev/container_operations.md]: > Running the command removes the container and any anonymous volumes joined

For clarity: Running the command removes the container and any anonymous volumes joined _to it_.

Comment (anchal-agrawal) [reply docs/user_doc/vic_app_dev/container_operations.md]: We should remove this sentence.

**Comment by alextopuzov** [comment]
@anchal-agrawal requested changes were implemented.

**Pull request merged**